src/modules/mana/sagas.ts

```diff
@@ -496,15 +494,16 @@ export const importWithdrawalErrors = {
   other: (msg: string) => formatImportWithdrawalError(msg),
 }
 
-function* handleImportWithdrawalRequest(action: ImportWithdrawalRequestAction) {
+export function* handleImportWithdrawalRequest(action: ImportWithdrawalRequestAction) {
   const {
     payload: { txHash },
   } = action
 
   try {
     const address: string | undefined = yield select(getAddress)
```

Comment:
If I'm not wrong, there should be a check here to see if the address is undefined or not. You can easily throw if that's the case and the catch will dispatch the failing action.
We'll also need a new test for it.

Reply:
Done